src/Historikk/Graf/LegendMedToggles/LegendMedToggles.tsx

```diff
@@ -25,11 +28,26 @@ export const LegendMedToggles: FunctionComponent<Props> = ({
         land: '',
     };
 
+    const orgnr = useOrgnr() || '';
+
     return (
         <CheckboxGroup
             legend="Velg linjer som skal vises i grafen"
             value={linjerSomSkalVises}
-            onChange={setLinjerSomSkalVises}
+            onChange={(value) => {
+                sendSykefraværsstatistikkIaMetrikk(orgnr);
+                if (value.length > linjerSomSkalVises.length) {
+                    // Brukeren har lagt til noe
+                    const verdiHuketAv = value.find((v) => linjerSomSkalVises.indexOf(v) === -1);
+                    sendCheckboxHuketAv(verdiHuketAv);
+                } else if (value.length < linjerSomSkalVises.length) {
+                    // Brukeren har fjernet noe
+                    const verdiHuketBort = linjerSomSkalVises.find((v) => value.indexOf(v) === -1);
```

Comment:
Måtte tenke meg om to ganger når jeg leste "huket av" og "huket bort". Kunne vi gått bort fra ordlyd som fokuserer på at det er en checkbox og heller fokusere på hva som legges til eller fjernes? Som f.eks. `lagtTil***` og `fjernet***` eller noe lignende?

Reply:
Jeg endret til lag til/fjernet.